error-reporting/api/diagnostics/Controllers/HomeController.cs

```diff
@@ -0,0 +1,45 @@
+﻿/*
+ * Copyright (c) 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using System.Net.Http;
+using System.Text;
+using ErrorReporting.Models;
+
+namespace ErrorReporting.Controllers
+{
+    public class HomeController : Controller
+    {
+        public HttpResponseMessage Index()

```

Comment:
I think you should specify the attribute on top of the action so people understand that they should make "GET" requests to get this action invoked.